recipes/ffmpeg/all/conanfile.py

```diff
@@ -295,6 +298,8 @@ def requirements(self):
             self.requires("libiconv/1.17")
         if self.options.with_freetype:
             self.requires("freetype/2.13.2")
+        if self.options.with_libxml2:
+            self.requires("libxml2/2.13.6")
```

Comment:
libxml2 should use a version range

Reply:
Fixed. I tested the build successfully. I would help if this would be merged quickly because currently not having this in master force me to export the recipe manually on my CI builder.